indexer-app/src/cmr/indexer/data/concepts/collection/opendata.clj

```diff
@@ -32,11 +32,15 @@
   "Returns the opendata related url for the given collection related url"
   [related-url]
   (let [{:keys [Description Type Subtype URL]} related-url
+        Format (get-in related-url [:GetData :Format])
+        get-data-mime-type (get-in related-url [:GetData :MimeType])
         MimeType (get-in related-url [:GetService :MimeType])
```

Comment:
Nit: the inconsistencies in case are bugging me. I think we should use Clojure convention outside of destructuring. So the fields in line 34 are ok, but `Format` should be `get-data-format` and `MimeType` below should be `get-service-mime-type`.